cmd/clusterctl/examples/aws/machines.yaml.template

```diff
@@ -4,7 +4,7 @@ items:
   - apiVersion: "cluster.k8s.io/v1alpha1"
     kind: Machine
     metadata:
-      generateName: aws-controlplane-
+      name: aws-controlplane-${CLUSTER_NAME}-0
```

Comment:
My apologies, I meant that we should be adding the cluster name to the instance through the actuator and not the machine template (which can be handled through a separate issue/PR)

Reply:
I guess this should still be ok for HA?

I agree we should change the Name in the actuator though

Reply:
> I guess this should still be ok for HA?

Yes, because for HA we will have 3 distinct machines for the control plane.